crates/adler32-rs/sgx-tests/enclave/Enclave.config.xml

```diff
@@ -0,0 +1,12 @@
+<!-- Please refer to User's Guide for the explanation of each field -->
+<EnclaveConfiguration>
+  <ProdID>0</ProdID>
+  <ISVSVN>0</ISVSVN>
+  <StackMaxSize>0x40000</StackMaxSize>
+  <HeapMaxSize>0x100000</HeapMaxSize>
+  <TCSNum>1</TCSNum>
+  <TCSPolicy>1</TCSPolicy>
```

Comment:
Set `TCSPolicy` to 0 to solve your issue.

Reply:
It works! Now the PR is ready for review~ @mssun